pkg/couchdb/ui/package.json

```diff
@@ -0,0 +1,52 @@
+{
+	"name": "@librocco/ui",
+	"version": "0.0.1",
+	"main": "./dist/index.js",
+	"scripts": {
+		"dev": "vite dev",
+		"build": "svelte-kit package",
+		"preview": "vite preview",
+		"check": "svelte-check --tsconfig ./tsconfig.json",
+		"check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
+		"lint": "prettier --check --plugin-search-dir=. . && eslint .",
+		"format": "prettier --write --plugin-search-dir=. .",
+		"test": "vitest",
+		"test:ci": "CI=true vitest run",
+		"storybook": "start-storybook -p 6006",
+		"build-storybook": "build-storybook",
+		"postinstall": "svelte-kit sync"
+	},
+	"devDependencies": {
+		"@librocco/svg": "workspace:*",
+		"@storybook/addon-actions": "^6.4.22",
+		"@storybook/addon-essentials": "^6.4.22",
+		"@storybook/addon-links": "^6.4.22",
+		"@storybook/addon-postcss": "^2.0.0",
+		"@storybook/addon-svelte-csf": "^2.0.3",
+		"@storybook/svelte": "^6.4.22",
+		"@sveltejs/adapter-auto": "next",
+		"@sveltejs/kit": "next",
+		"@testing-library/svelte": "~3.1.3",
+		"@typescript-eslint/eslint-plugin": "^5.27.0",
+		"@typescript-eslint/parser": "^5.27.0",
+		"eslint": "^8.16.0",
+		"eslint-config-prettier": "^8.3.0",
+		"eslint-plugin-svelte3": "^4.0.0",
+		"postcss": "~8.4.13",
+		"prettier": "^2.6.2",
+		"prettier-plugin-svelte": "^2.7.0",
+		"svelte": "^3.44.0",
```

Comment:
Go over the deps and remove anything `svelte` and `storybook` related as we won't be using those

Reply:
Also postcss, tailwind

Reply:
Also tslib